routes/applications.js

```diff
@@ -377,4 +381,137 @@ router.delete(applicationIdRoute, authentication.authorized, function (req, res,
     });
 });
 
+/**
+ * @api {put} /applications/look/:prefix Changes the application look field.
+ * @apiName PutApplicationLook
+ * @apiGroup Applications
+ *
+ * @apiParam {String} prefix Application prefix.
+ *
+ * @apiPermission admin
+ *
+ * @apiParamExample {json} Request-Example:
+ *      {
+ *          "key":"_id",
+ *          "user": "dev"
+ *          "resource":"id1",
+ *          "methods":["post","put"],
+ *          "url":"/url/*"
+ *      }
+ *
+ * @apiSuccess(200) Success.
+ *
+ * @apiSuccessExample Success-Response:
+ *      HTTP/1.1 200 OK
+ *      {
+ *          "_id": "559a447831b7acec185bf513",
+ *          "name": "Gleaner App.",
+ *          "prefix": "gleaner",
+ *          "host": "localhost:3300",
+ *          "anonymous": [],
+ *          "look":[{
+ *              "key":"_id",
+ *              "permissions":{
+ *                 "dev":["id1","id2"]
+ *               },
+ *              "methods":["post","put"],
+ *              "url":"/url/*"
+ *          }],
+ *          "timeCreated": "2015-07-06T09:03:52.636Z"
+ *      }
+ *
+ * @apiError(400) InvalidApplicationId You must provide a valid application name.
+ *
+ * @apiError(400) ApplicationNotFound No application with the given application id exists.
+ *
+ */
+router.put('/look/:prefix', authentication.authorized, function (req, res, next) {
+    req.app.db.model('application').findByPrefix(req.params.prefix, function (err, results) {
+        if (err) {
+            return next(err);
+        }
+
+        var applicationId = results._id;
+        var query = {
+            _id: applicationId
+        };
+        var update = {
+            $set: {}
+        };
+
+        var newLook = results.look;
+        var added = false;
+        var error;
```

Comment:
I think this commit should be updated since it completely changes in the last commit: https://github.com/e-ucm/a2/pull/94/commits/f5875dc2fd1e2f563dad09b6404380761b89edc6